tests/core/test_controldata.py

```diff
@@ -0,0 +1,121 @@
+"""Tests for all control modules."""
+import pytest
+
+from ansys.meshing.prime.params.primestructs import SizingType
+
+
+@pytest.mark.skip(reason="Periodic control is broken in Prime server version <= 23.1")
+def test_periodic_control(initialized_model_elbow):
+    """Tests the methods and properties from ControlData class and control classes."""
+    model, _ = initialized_model_elbow
+    controldata = model.control_data
```

Comment:
Can we rename this variable `control_data` if possible or will it create any conflicts with other variables sharing a similar name?
```suggestion
    control_data = model.control_data
```